js/components/Textcolor.js

```diff
@@ -0,0 +1,15 @@
+const React = require("react");
+
+class Textcolor extends React.Component {
+	render() {
+		const size = this.props.size;
+		const color = this.props.color;
```

Comment:
This could be represented as a destructured object (`const { size, color } = this.props;`) but no need to change, just thought I'd note.